content/administration/upgrade.rst

```diff
@@ -225,8 +225,8 @@ Service Level Agreement
 What is covered by the Enterprise Licence?
 ------------------------------------------
 
-Databases hosted on Odoo’s Cloud platforms (Saas and Odoo.sh) or On-Premise (Self-Hosting) enjoy the
-following services at all times.
+Databases hosted on Odoo’s Cloud platforms (Odoo Online and Odoo.sh) or On-Premise (Self-Hosting)
```

Comment:
```suggestion
Databases hosted on Odoo Online platforms (Odoo Online and Odoo.sh) or On-Premise (Self-Hosting)
```

Reply:
Nope, actually, we already have something called "Odoo Cloud" (I know, it would have been another mess to distinguish that). It is about Odoo Cloud platforms (the databases that we host: both Odoo Onlilne and Odoo.sh).

But well spotted :)